proto/eth/v2/beacon_block.proto

```diff
@@ -228,7 +228,7 @@ message BeaconBlockBodyBellatrix {
   bytes graffiti = 3 [(ethereum.eth.ext.ssz_size) = "32"];
 
   // Block operations
-  // Refer to spec constants at https://github.com/ethereum/consensus-specs/blob/dev/specs/core/0_beacon-chain.md#max-operations-per-block
+  // Refer to spec constants at https://github.com/ethereum/consensus-specs/blob/dev/specs/phase0/beacon-chain.md#max-operations-per-block
```

Comment:
Shouldn't this be bellatrix?

Reply:
These should all be phase 0 because the operations aren't redefined in the new fork. Only the newer ones are added. I'm tempted to just remove the spec reference, it seems like they are adding more confusion than good

Reply:
At least put it in the PR description so we can cross check the values. I was hoping to find the link that shows what these values are supposed to be and then I could approve with a review that the PR is correct to spec.

Reply:
Nevermind, the object links in the description are good.